ipywidgets/.npmignore

```diff
@@ -6,6 +6,7 @@ docs/man/*.gz
 docs/source/api/generated
 docs/gh-pages
 static/components/
+static/notebook/js/
```

Comment:
Looking at this more, we shouldn't have done this.  When we do the small split, **no** javascript will live in ipywidgets.  The javascript, included notebook specific javascript, should be in the npm package available to install by ipywidgets or w/e.  I'll re-include this in #336